pkg/ingress/ingress_test.go

```diff
@@ -89,6 +87,8 @@ func TestIngressNetworkingAPI(t *testing.T) {
 }
 
 func TestIngressExtensionAPI(t *testing.T) {
+	t.Skip("test skipped after migrating to Operator SDK 1.18.2")
```

Comment:
@rubenvp8510 I wasn't able to understand what this test is actually trying to accomplish. From what I understood, there is (was?) a conversion between a legacy ingress type into a newer type. Could you please confirm that this is still happening? Or perhaps you can spot what I did wrong in this test, causing it to fail...

Reply:
The conversion was the intention of this PR.  Is this test failing with the SDK update? This was introduced for maintain backward compatibility as an step to migrate to the newer networking.k8s.io/v1beta1 API.

Reply:
The problem is: at the time I did the test I created an object `Ingress` with the `namespace` and `name` that match the object I want to update, then send it as an update to the API. That is an incorrect approach, but for some reason it wasn't complain.

Now you fixed it, the problem with your approach is that you are using the new `Network` API to get the ingress resource, but it doesn't exist because it was created by the old `extension` API. Indeed is what this test is testing that we are using the old API if the `ingress-api` flag is set to use it.


I'll change the following lines 
```

updatedIngress := &netv1beta.Ingress{}
	err = cl.Get(context.Background(), types.NamespacedName{Name: ingress.GetName(), Namespace: ingress.GetNamespace()}, updatedIngress)
	require.NoError(t, err)


```
With this:

```
	updatedIngressExt := &extv1beta.Ingress{}

	err = cl.Get(context.Background(), types.NamespacedName{Name: ingress.GetName(), Namespace: ingress.GetNamespace()}, updatedIngressExt)
	require.NoError(t, err)
	updatedIngress := ingressClient.fromExtToNet(*updatedIngressExt)

	updatedIngress.Spec.Backend.ServiceName = "updated-srv"
	err = ingressClient.Update(context.Background(), &updatedIngress)
	require.NoError(t, err)
```

Said that, you haven't broken anything :) So if you want I can fix the test in a following PR.

Reply:
It now passes, thanks!